Jbarget/oxymore
# adds Facebook pixel code

### What changes have you made?
- added a Facebook pixel code to the `index.html` so that traffic coming from paid Facebook ads can be measured
- updated a couple of meta tags 

### Which issue(s) does this PR fix?

Fixes #354

### Screenshots (if there are design changes)

### How to test
- go to index.html
- make sure there is a Facebook pixel at the bottom of the header section, just above the closing head tag
